### ISSUE TITLE
TypeError:unsupported operand type(s) for |: 'type' and 'NoneType'

### ISSUE THREAD

**Issue by alphacheng** [edsaac/stpyvista#18]
```
import pyvista as pv
import streamlit as st
from stpyvista import stpyvista

st.title("A cube")
st.info("""Code adapted from https://docs.pyvista.org/user-guide/jupyter/pythreejs.html#scalars-support""")

## Initialize a plotter object
plotter = pv.Plotter(window_size=[400,400])

## Create a mesh with a cube 
mesh = pv.Cube(center=(0,0,0))

## Add some scalar field associated to the mesh
mesh['myscalar'] = mesh.points[:, 2] * mesh.points[:, 0]

## Add mesh to the plotter
plotter.add_mesh(mesh, scalars='myscalar', cmap='bwr')

## Final touches
plotter.view_isometric()
plotter.background_color = 'white'

## Send to streamlit
stpyvista(plotter, key="pv_cube")
```
Any guidance on resolving this issue would be greatly appreciated :)
Additional Information:
python version: 3.9.13
pyvista version: 0.43.4
Streamlit version: 1.32.2
stpyvista version: 0.0.14
panel version: 0.13.1

**Comment by edsaac** [comment]
`stpyvista` 0.0.14 was tested for python>=3.10 only. Upgrading should take care of your issue. The other option is to upgrade `stpyvista` to version 0.0.16  where I fixed that type hint to work with python 3.9.

**Comment by alphacheng** [comment]
Thank you for your reply. I will try it out
